### PR TITLE
REL: 0.4.2

### PR DESCRIPTION
**Pull request by sappelhoff**
following with some more cleanups after #302

for a 0.4.2 patch release

### CODE REVIEW

File: examples/plot_autoreject_workflow.py [hunk 1]
```diff
@@ -41,7 +41,11 @@
 dataset = 'ds002778'  # The id code on OpenNeuro for this example dataset
 subject_id = 'pd14'
 
-target_dir = op.join(op.dirname(autoreject.__file__), '..', 'examples')
+target_dir = os.path.join(
+    os.path.dirname(autoreject.__file__), '..', 'examples', dataset)
+if not os.path.isdir(target_dir):
```
Comment (jasmainak) [inline examples/plot_autoreject_workflow.py:46]: openneuro doesn't make it if it doesn't exist?

Comment (sappelhoff) [reply examples/plot_autoreject_workflow.py:46]: it takes the `dataset` parameter as a dir name and creates it if it doesn't exist, but only if no `target_dir` argument is passed: https://github.com/hoechenberger/openneuro-py/blob/c6f994cebe67c78e3208f34ac4e7d8753fce506a/openneuro/_download.py#L696-L699

Comment (sappelhoff) [reply examples/plot_autoreject_workflow.py]: I simplified it a bit in https://github.com/autoreject/autoreject/pull/303/commits/eba75a73724f8eccd34c4d47a32d13ff0acef1cb

**Comment by sappelhoff** [comment]
I also added a new repository secret: 
image


This will make the circleci-artifact-redirector work again. Following instructions from https://github.com/larsoner/circleci-artifacts-redirector-action

**Comment by sappelhoff** [comment]
also adjusted branch protection rules for "required CI checks" to use the Py versions that we are using now.

interestingly, the dropdown menu for versions does not work anymore, neither on this PR, nor on master.

Try here:

- master: https://autoreject.github.io/dev/index.html
- this PR: https://output.circle-artifacts.com/output/job/d505096a-7f44-4bb8-b3e0-ee6cf173596f/artifacts/0/html/index.html

any idea why @larsoner ?

**Comment by larsoner** [comment]
Yes I would make the switch :)

**Comment by sappelhoff** [comment]
there we go! 

https://pypi.org/project/autoreject/0.4.2/

🎉

**Comment by jasmainak** [comment]
Thanks @sappelhoff !!

**Pull request merged**